database/redis/selfstate.go

```diff
@@ -36,17 +38,17 @@ func (connector *DbConnector) GetChecksUpdatesCount() (int64, error) {
 func (connector *DbConnector) GetRemoteChecksUpdatesCount() (int64, error) {
 	c := *connector.client
 	ts, err := c.Get(connector.context, selfStateRemoteChecksCounterKey).Int64()
-	if err == redis.Nil {
+	if errors.Is(err, redis.Nil) {
 		return 0, nil
 	}
 	return ts, err
 }
 
-// GetRemoteChecksUpdatesCount return remote checks count by Moira-Checker
+// GetPrometheusChecksUpdatesCount return remote checks count by Moira-Checker
```

Comment:
Можно еще сам комментарий подправить:
```suggestion
// GetPrometheusChecksUpdatesCount return prometheus checks count by Moira-Checker
```

Reply:
Кажется, я эту штуку полностью переделаю сейчас. Так что мб не стоит трогать

Reply:
не трогаю